specs/LaunchingRegisteredProtocols.md

```diff
@@ -0,0 +1,264 @@
+# Examples
+
+## Win32 C++
+    
+```cpp 
+
+CHECK_FAILURE(m_webView->add_LaunchingRegisteredProtocol(
+    Callback<ICoreWebView2LaunchingRegisteredProtocolEventHandler>(
+        [this](
+            ICoreWebView2* sender,
+            ICoreWebView2LaunchingRegisteredProtocolEventArgs* args) {
+                wil::unique_cotaskmem_string uri;
+                CHECK_FAILURE(args->get_Uri(&uri));
+                if (wcscmp(uri.get(), L"calculator://") == 0)
+                {
+                    CHECK_FAILURE(args->put_Handled(TRUE));
+                    // If this matches our desired protocol, then suppress the
+                    // popup dialog.
+                }
+                else
+                {
+                    CHECK_FAILURE(args->put_Handled(FALSE));
+                    // Otherwise allow the popup dialog, and allow the user to decide
+                    // whether or not to allow the protocol to launch.
+                }
+            return S_OK;
+        })
+        .Get(),
+    &m_launchingRegisteredProtocolToken));
+
+CHECK_FAILURE(frame->add_LaunchingRegisteredProtocol(
+    Callback<ICoreWebView2FrameLaunchingRegisteredProtocolEventHandler>(
+        [this](
+            ICoreWebView2Frame* sender,
+            ICoreWebView2LaunchingRegisteredProtocolEventArgs* args) {
+                wil::unique_cotaskmem_string uri;
+                CHECK_FAILURE(args->get_Uri(&uri));
+                if (wcscmp(uri.get(), L"calculator://") == 0)
+                {
+                    // If this matches our desired protocol, then suppress the
+                    // popup dialog.
+                    CHECK_FAILURE(args->put_Handled(TRUE));
+                }
+                else
+                {
+                    // Otherwise revoke permissions previously granted to this protocol 
+                    // from this origin. This will trigger the dialog to appear to allow the user
+                    // to confirm or deny the protocol launch.
+                    CHECK_FAILURE(args->put_RevokeProtocolPermissionsPerOrigin(TRUE));
+                }
+                return S_OK;
+        })
+        .Get(),
+    &m_frameLaunchingRegisteredProtocolToken));
+    
+``` 
+
+## .NET and WinRT
+
+```c #
+
+void WebView_LaunchingRegisteredProtocol(object target, CoreWebView2LaunchingRegisteredProtocolEventArgs e)
+{
+    if (e.Uri == "calculator:///") 
+    {
+        // If this matches our desired protocol, then suppress the popup dialog.
+        e.Handled = true;
+    }
+    else 
+    {
+        // Otherwise allow the popup dialog, and allow the user to decide 
+        // whether or not to allow the protocol to launch.
+        e.Handled = false;
+    }
+    webView.CoreWebView2.FrameCreated += (sender, args) =>
```

Comment:
It looks weird to subscribe to FrameCreated inside the LaunchingRegisteredProtocol event handler. Also webView variable doesn't look like it would be defined anywhere?